serverless.yaml

```diff
@@ -14,6 +14,7 @@ custom:
   oauthRedirect: ${opt:oauthRedirect, 'http://localhost'}
   config: ${file(serverless_config.json)}
   integrationTransformPath: '${self:service}.integration-transform.${self:custom.region}.${self:custom.stage}.url'
+  integrationTransformAwsRegionPath: '${self:service}.integration-transform.${self:custom.region}.${self:custom.stage}.awsRegion'

```

Comment:
Use `/` instead of `.` in the parameter name.
SSM has the concept of hierarchical parameters and prefixes. It helps keep params organized and makes it easier to manage permissions.

Also, maybe it's not worth including `${self:custom.region}` in the param name since params only exist in a region anyways.

https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-paramstore-su-create.html

https://aws.amazon.com/blogs/mt/organize-parameters-by-hierarchy-tags-or-amazon-cloudwatch-events-with-amazon-ec2-systems-manager-parameter-store/

Reply:
Updated to `/fhir-service/integration-transform/<STAGE>/url` and `/fhir-service/integration-transform/<STAGE>/awsRegion`